src/Components/Body.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { render, fireEvent, waitFor, getByAltText, mockClear, getByPlaceholderText, GetByRole, screen } from '@testing-library/react';
```

Comment:
Im thinking we could remove a couple of the unused imports on this one. Thoughts?